packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -155,6 +156,7 @@
 
     .tab-menu:focus {
       background-color: var(--terra-tabs-structural-focus-background-color, #fff);
+      color: #000;
```

Comment:
Let's set this to color: `var(--terra-tabs-structural-focus-color, inherit);`. This will inherit the parent elements color which is likely inherited from base. Then we should be good to just have 1 instance of color in this selector.